omnibus/config/projects/supermarket.rb

```diff
@@ -29,8 +29,11 @@
 build_iteration 1
 
 # NOTE: see the omnibus-supermarket cookbook gemfile for controlling the infra client version
-override :postgresql, version: '9.3.25'
+override :postgresql, version: '13.3'
```

Comment:
Let's go to 13.4 now that it's out and has a CVE fix.

Reply:
This has been handled in other way by using own postgres software definition in supermarket.